subsys/net/lib/dns/dns_cache.c

```diff
@@ -97,8 +97,8 @@ int dns_cache_remove(struct dns_cache *cache, char const *query)
 	return 0;
```

Comment:
Please remove the `subsys: ` from commit subject, it is redundant information and not needed. So start the commit by `net:`